src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,28 @@
 package mate.jdbc;
 
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Manufacturer;
+
 public class Main {
-    public static void main(String[] args) {
+    public static final Injector injector = Injector.getInstance("mate.jdbc");
 
+    public static void main(String[] args) {
+        ManufacturerDao manufacturerDao =
+                (ManufacturerDao) injector.getInstance(ManufacturerDao.class);
+        Manufacturer porscheManufacturer = new Manufacturer("Porsche", "Germany");
+        Manufacturer mustangManufacturer = new Manufacturer("Mustang", "USA");
+        manufacturerDao.create(porscheManufacturer);
+        manufacturerDao.create(mustangManufacturer);
+        System.out.println(manufacturerDao.getAll());
+        System.out.println(manufacturerDao.get(9L));
```

Comment:
не зовсім зрозуміло чому 9L, давай використаємо ID уже створено раніше обєкту.
```suggestion
        System.out.println(manufacturerDao.get(mustangManufacturer.getId()));
```